Resources/Prototypes/Recipes/Reactions/pyrotechnic.yml

```diff
@@ -42,7 +42,8 @@
     devastationRange: 0
     heavyImpactRange: 0
     lightImpactRange: 2
-    scaled: false
+    scaled: true
+    maxScale: 10 # explosion strength stops scaling at 10 chlorine + 30 fluorine
```

Comment:
Unless I'm misremembering how old explosion code works, a max scale of 10 will mean you can get a radius-10 explosion out of this.  The max should probably just be 1, with a larger reagent cost

Reply:
Done.